rickchurch/templates/base.html

```diff
@@ -5,6 +5,7 @@
   {% block head %}
   <title>rickchurch</title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="/static/styles.css">
```

Comment:
```suggestion
  <link rel="stylesheet" href="{{ url_for('static', path='/styles.css') }}">
```
You should use `url_for`, so that we can specify the static directory in `church.py` without any hard-coding.
(source: https://fastapi.tiangolo.com/advanced/templates/#templates-and-static-files)

Reply:
Ok, I'll switch to this but it'll be a bit of a pain for inline images